helpers/handlePeripheryContractDeploy.ts

```diff
@@ -0,0 +1,21 @@
+import { deployContract } from './deployContract'
+
+export interface handlePeripheryContractDeployConfig {
+  prizeDistributor: string,
+  prizeSplitStrategy: string,
+  reserve: string,
+}
+
+export async function handlePeripheryContractDeploy(deploy: Function, deployer: string, config: handlePeripheryContractDeployConfig = {
+  prizeDistributor: '',
+  prizeSplitStrategy: '',
+  reserve: ''
+}) {
+  const prizeFlushResult = await deployContract(deploy, 'PrizeFlush', deployer, [deployer, config.prizeDistributor, config.prizeSplitStrategy, config.reserve])
+
+  return {
+    prizeFlushResult: prizeFlushResult,
+  }
+}
+
+export default handlePeripheryContractDeploy
```

Comment:
hmm....

`handlePeripheryContractDeploy` is only called in one place.  This file adds *a lot* of code without any real benefit other than consistency in approach.  Less code is better!  Basically a maxim.

✂️

Reply:
I would argue consistency with approach is pretty important. It's weird to switch in the middle of execution.

Reply:
You've added an extra file and 21 lines of code for no additional benefit.

This is way too much overhead.